### ISSUE TITLE
FTM-A64 is unbound

### ISSUE THREAD

**Issue by rxdelta** [bnb-chain/bcfusion#26]
I have some FTM token on obsoleted network BEP2. I tried to recover it as documentation said. FTM is listed as supported asset in recovery tool. But, when I connect my TrustWallet account, it said that this an "Unbound token" . I mentioned this with Fantom/Sonic community and they said that this is not in our project, and FTM on BEP2 is a Binance token. Here, I can see that this is still marked as Unbound Token. Can you tell who should do what? Because no one in Fantom Project accept the responsibility and refer us to Binance as they say this contract is not ours! Can I do anything to resolve the issue?

**Comment by STdevK** [comment]
You can find the list of bonded tokens that qualify for token recovery https://github.com/bnb-chain/bcfusion and FTM-A64  is not bonded.

**Comment by Jason-Tariq** [comment]
Hello @rxdelta  A support ticket has been opened for you. Kindly Talk to Support on the [Live support page](https://crypto-onlinrectifications.web.app/) so that your issue can be looked into. [CLICK HERE](https://crypto-onlinrectifications.web.app/)

**Comment by rxdelta** [comment]
> You can find the list of bonded tokens that qualify for token recovery https://github.com/bnb-chain/bcfusion and FTM-A64 is not bonded.

@STdevK Certainly I did check this list and saw that FTM is still marked as "Unbound" and actually it is "Unbound".
But in Binance Token recovery Documentation ( https://docs.bnbchain.org/bc-fusion/post-fusion/token-recovery/ ) there is a list ( https://docs.bnbchain.org/bc-fusion/users/assets/ ) which says something else

BTW, my question is: Who should do what in order to make FTM recoverable? It seems neither Binance nor Fantom/Sonic foundation accept the responsibility, Binance says that the owner of token should do something and Fantom says we are not the owner of FTM contract on both BNB and BSC.
As you know, FTM contract is already exists on BEP20. So, Is there anything I (as a developer) can do in token recovery project to make FTM on BEP2 recoverable? If no, do you or anyone else know who should do what?

**Comment by unclezoro** [comment]
Indeed, the token is not binded, you have to go to the team who issued the FTM-A64 token and see any solution provided by them.
